### PR TITLE
[v6r20] web compiler support for extjs6

### PR DESCRIPTION
**Pull request by zmathe**
BEGINRELEASENOTES
*FrameworkSystem
FIX: add extjs6 support to the web compiler
ENDRELEASENOTES

### CODE REVIEW

File: FrameworkSystem/Client/WebAppCompiler.py [hunk 1]
```diff
@@ -235,7 +247,7 @@ def run(self):
     gLogger.verbose(" IN file written to %s" % inFile)
 
     cmd = ['sencha', '-sdk', self.__sdkPath, 'compile', '-classpath=%s' % ",".join(self.__classPaths),
-           '-debug=%s' % self.__debugFlag, 'page', '-yui', '-input-file', inFile, '-out', outFile]
```
Comment (fstagni) [inline FrameworkSystem/Client/WebAppCompiler.py]: you don't want debug?

Comment (zmathe) [reply FrameworkSystem/Client/WebAppCompiler.py]: It is supposed to be used for printing the debug messages, which does not help in production...

**Comment by fstagni** [comment]
Do we need to merge this in v6r20? Why not integration?

**Comment by zmathe** [comment]
yes. Because this module will be used by lb-run LHCbDIRAC/prod dirac-distribution -r ...
So, the web compiler needs to be released.

**Comment by zmathe** [comment]
yes. Because this module will be used by lb-run LHCbDIRAC/prod dirac-distribution -r ...
So, the web compiler needs to be released.

**Pull request merged**